tests/kernels/utils.py

```diff
@@ -0,0 +1,21 @@
+"""Kernel test utils"""
+
+import pytest
+
+STR_BACKEND_ENV_VAR: str = "VLLM_ATTENTION_BACKEND"
+STR_FLASH_ATTN_VAL: str = "FLASH_ATTN"
+STR_INVALID_VAL: str = "INVALID"
+
+
+def override_backend(mpatch: pytest.MonkeyPatch, backend_name: str) -> None:
```

Comment:
nit: please consider renaming to override_backend_env_variable.

Reply:
Fixed in latest push